### ISSUE TITLE
Error reading input file

### ISSUE THREAD

**Issue by schoettl** [Moduland/csv2vcf#2]
When I open a csv file (comma- or semicolon-separated, UTF-8 or ASCII), I only get the error "error in reading input file". No useful debug messages are printed on the terminal.

Also, the command line version does not work as it should:
```
python3 -m csv2vcf file.csv
```
shows the UI in the same way as without the filename. It would be better if it just converts file.csv to file.vcf.

**Comment by sepandhaghighi** [comment]
@schoettl Hi
Thanks for your comment ;-)
Would you please give us more information about your `.csv` file?

**Comment by schoettl** [comment]
Hi, this is my csv file:
```
name,home,work,mobile,combined
Essen,,08000 1234560,,08000 1234560
```

I noticed that the error does not appear when I omit the header line.

However, without the header line, it shows "0 VCF File Generated In 00 days, 00 hour, ..." and the target folder is empty.

Also, after both message dialogs, the program cannot be closed and crashes after I press ctrl-c and switch window. I cannot close a window because the window disappears when I click "OK" in the dialog.

You should really consider implementing my suggestions because there seem to be no competing command line conversion tools except [csv2vcf](https://gist.github.com/mridah/5d8aac6b4150373b1be57dd4e12b7fb0) which is python2 and cannot handle different phone number types. But the field assignments are easily configurable.

**Comment by schoettl** [comment]
Right! I forgot to use your specified input format. I used to have it but then I had to change the format to use the other converter tool.

**Comment by sepandhaghighi** [comment]
@schoettl Thanks again for your comments and suggestions ;-)
We didn't update this tool for about 2 year, but now I have a plan to release a new version in next few months.
